### DISCUSSION TITLE
Searching within "Releases" is broken

### DISCUSSION THREAD

**Discussion by GiacoCorsiglia** [community/community#114803] [Repositories]
### Select Topic Area

Bug

### Body

When searching for a keyword on the "Releases" page, many releases matching that keyword are NOT included in the results.  There doesn't seem to be a pattern to which releases are actually included, or why.  But most results are NOT included, rendering the search feature useless.

Example: [this search](https://github.com/nextauthjs/next-auth/releases?q=dynamodb&expanded=true).  

I am searching for "dynamodb" on the Releases page of the [nextauthjs/next-auth](https://github.com/nextauthjs/next-auth) repository.  The search returns 7 total results (only one page of results).  However, there are many other releases that include "dynamodb" in the title for that project.  For example, [this release](https://github.com/nextauthjs/next-auth/releases/tag/%40auth%2Fdynamodb-adapter%401.3.3) is not listed in the search results.

Thanks!

**Comment by akeller** [comment]
I'm having a similar awful experience the [camunda-platform repo](https://github.com/camunda/camunda-platform/releases). The search highlights any characters in the keyword. It doesn't jump to the first highlighted instance either, which I believe was the previous behavior. Ctrl + F on the page is better for me now since there is no filtering, only highlighting.

I was looking for "3.13" and it highlighted "3", but if I search "Optimize" it highlights the whole word. 🤷‍♀️

**Comment by A-Stapleton** [comment]
This happens on https://github.com/Azure/azure-sdk-for-net/releases as well where you search for `Azure.Identity` (or anything else) and it shows anything with Azure in it completely ignoring the rest.

**Comment by trevor-scheer** [comment]
The false positive matches are totally bananas. Another example - see the highlights across the page.
https://github.com/graphql/graphql-js/releases?q=v16.6.0&expanded=true

What I'd _really_ like is to filter the releases to a specific semver range.
